characters/instruction-following/Galactica Cite.yaml

```diff
@@ -0,0 +1,4 @@
+user: ""
+bot: "[START_REF]"
+turn_template: "<|user-message|> <|bot|><|bot-message|>\n\n"
+context: ""
```

Comment:
NL@EoF missing on a bunch of these files (can see GitHub's lil red icon to highlight that)